website/content/en/docs/cli/operator-sdk_alpha.md

```diff
@@ -27,4 +30,5 @@ Commands in alpha stage. These APIs are not stable and may be removed at any tim
 
 * [operator-sdk](../operator-sdk)	 - 
 * [operator-sdk alpha config-3alpha-to-3](../operator-sdk_alpha_config-3alpha-to-3)	 - Convert your PROJECT config file from version 3-alpha to 3
+* [operator-sdk alpha config-gen](../operator-sdk_alpha_config-gen)	 - Generate configuration for controller-runtime based projects
```

Comment:
I think we might hide this one since it is in a very premature stage

Reply:
E.g done in the past : https://github.com/operator-framework/operator-sdk/blob/v0.18.x/cmd/operator-sdk/cli/legacy.go#L94-L100

Reply:
Done